src/format.rs

```diff
@@ -181,6 +181,26 @@ impl PayloadParams {
         self.spec
     }
 
+    /// Whether the payload use the TWCC feedback mechanic.
+    pub fn fb_transport_cc(&self) -> bool {
+        self.fb_transport_cc
+    }
+
+    /// Whether the payload uses NACK to request resends.
+    pub fn fb_nack(&self) -> bool {
+        self.fb_nack
+    }
+
+    /// Whether the payload uses the PLI (Picture Loss Indication) mechanic.
+    pub fn fb_pli(&self) -> bool {
+        self.fb_pli
+    }
+
+    /// Whether the payload uses the FIR (Full Intra Request) mechanic.
+    pub fn fb_fir(&self) -> bool {
+        self.fb_fir
+    }
+
```

Comment:
How is this useful without also being able to set these values?

Not saying we shouldn't do it, just want to understand.

Reply:
Currently we just generate default codecs and then read out the values and convert to our internal format. But I agree for more flexibility it would be useful to set them so one can customize the codecs fully.

Let me add that.

Reply:
@algesten probably fine to have a set api for this and not expading the constructor, but let me know what you think.

Reply:
Either way it creates a bit of a problem, because we don't expect the sdp API to set any of this (since it is negotiated).

I think ultimately we want a `PayloadParamBuilder` that sits under `rtp` or some such (where I expose lower level stuff). This is to make clear on a code level, that we don't expect people to normally use this.

However for this first iteration, I think setters are enough (I haven't made my mind up on how to deal with this exactly).